test/test_neatjson-opal.rb

```diff
@@ -0,0 +1,225 @@
+require_relative 'neatjson'
```

Comment:
To be honest, I did not quickly manage to include the same test file. So I copied it into test_neatjson-opal.rb. I guess we need a separate test main script, but the testcases should not be  written differently. I even wonder if the testcases could be written in an extra JSON file and used for javascript, ruby and opalrb.